specification/search/data-plane/Microsoft.Azure.Search.Service/stable/2019-05-06/searchservice.json

```diff
@@ -4518,6 +4518,10 @@
       "properties": {
         "@odata.type": {
           "type": "string"
+        },
```

Comment:
Please add the new `name` property to at least some of the skillset examples as part of this PR.